spec/requests/api/v1/topics/view_spec.rb

```diff
@@ -0,0 +1,28 @@
+## Get all topics
```

Comment:
Remove these kinds of comments, due to Ruby conventions you know which kind of endpoint you are testing. And maybe instead of `view_spec.rb` it should be `index_spec.rb`, wdyt?